decidim-design/app/helpers/decidim/design/cards_helper.rb

```diff
@@ -126,53 +142,442 @@ def cards_sections
-                  { name: "Process G",
-                    url: "https://github.com/decidim/decidim/blob/develop/decidim-participatory_processes/app/cells/decidim/participatory_processes/process_g_cell.rb", usage: "https://github.com/decidim/decidim/blob/develop/decidim-participatory_processes/app/views/decidim/participatory_processes/participatory_processes/_collection.html.erb#L5" },
-                  { name: "Process Group G",
-                    url: "https://github.com/decidim/decidim/blob/develop/decidim-participatory_processes/app/cells/decidim/participatory_processes/process_group_g_cell.rb", usage: "https://github.com/decidim/decidim/blob/develop/decidim-participatory_processes/app/views/decidim/participatory_process_groups/_participatory_process_group.html.erb#L1" },
-                  { name: "Proposal G", url: "https://github.com/decidim/decidim/blob/develop/decidim-proposals/app/cells/decidim/proposals/proposal_g_cell.rb",
-                    usage: "https://github.com/decidim/decidim/blob/develop/decidim-proposals/app/views/decidim/proposals/proposals/_proposals.html.erb#L15" }
-                )
-              }
-            ]
+            id: "source_code",
+            contents: source_contents
           }
         ]
       end
       # rubocop:enable Layout/LineLength
 
+      # rubocop: disable Metrics/CyclomaticComplexity
+      # rubocop: disable Metrics/PerceivedComplexity
+      def source_contents
+        dummy_resource = DummyClass.new(
+          id: 1000,
+          organization: current_organization,
+          title: "Dummy resource title",
+          description: "Dummy resource description"
+        )
+
+        contents = [
+          { values: section_subtitle(title: "Generic cards") },
+          cell_table_item("Card L", { cell: "decidim/card_l", args: [dummy_resource], call_string: 'cell("decidim/card_l", _RESOURCE_)' }),
+          cell_table_item("Card G", { cell: "decidim/card_l", args: [dummy_resource], call_string: 'cell("decidim/card_g", _RESOURCE_)' }),
+          cell_table_item("Card S", { cell: "decidim/card_l", args: [dummy_resource], call_string: 'cell("decidim/card_s", _RESOURCE_)' })
+        ]
+
+        if Decidim.module_installed?(:accountability) && (resource = Decidim::Accountability::Result.last).present?
+          contents += [
+            { values: section_subtitle(title: "Accountability") },
+            cell_table_item(
+              "Result L",
+              {
+                cell: "decidim/accountability/result_l",
+                args: [resource],
+                call_string: [
+                  "card_for(_RESOURCE_)",
+                  "card_for(_RESOURCE_, size: :l)",
+                  'cell("decidim/accountability/result", _RESOURCE_)',
+                  'cell("decidim/accountability/result", _RESOURCE_, size: :l)',
+                  'cell("decidim/accountability/result_l", _RESOURCE_)'
+                ]
+              }
+            )
+          ]
+        end
```

Comment:
Great change by actually checking if the module is installed! 

Let's try to reduce the number of lines in this method by making the typical refactor of extracting method:
```suggestion
        contents += contents_for_accountability
```

And then the method (that should be private) is this one:
```ruby
      def contents_for_accountability
        return unless Decidim.module_installed?(:accountability) && (resource = Decidim::Accountability::Result.last).present?

        [
          { values: section_subtitle(title: "Accountability") },
          cell_table_item(
            "Result L",
            {
              cell: "decidim/accountability/result_l",
              args: [resource],
              call_string: [
                "card_for(_RESOURCE_)",
                "card_for(_RESOURCE_, size: :l)",
                'cell("decidim/accountability/result", _RESOURCE_)',
                'cell("decidim/accountability/result", _RESOURCE_, size: :l)',
                'cell("decidim/accountability/result_l", _RESOURCE_)'
              ]
            }
          )
        ]
      end
```

We should implement this for each of the modules of course